common4j/src/main/com/microsoft/identity/common/java/nativeauth/commands/parameters/AcquireTokenNoFixedScopesCommandParameters.java

```diff
@@ -36,6 +36,7 @@
 import java.net.URISyntaxException;
```

Comment:
Do we still need this file if `AcquireTokenNoFixedScopesCommand` is being removed?

Reply:
The file has been removed.